hackclub.com.yaml

```diff
@@ -1142,6 +1146,10 @@ f5qsmeu4qwef4yvo2p6e4a4vrspzdqdc._domainkey.epochvt:
   ttl: 600
   type: CNAME
   value: f5qsmeu4qwef4yvo2p6e4a4vrspzdqdc.dkim.amazonses.com.
+fa:
+  - ttl: 1
+    type: CNAME
+    value: cname.vercel-dns.com.
```

Comment:
What is this record?

Reply:
Hi, I hope you're having a wonderful day!

It's for the Persian Translation of Hack Club that I've made.
You can view my pull request here: https://github.com/hackclub/global/pull/116

Reply:
In the readme it mentioned that to get the subdomain going I'd have to create a pull request in the dns repository!

Reply:
![image](https://github.com/user-attachments/assets/ed5016bf-1f78-467d-aea1-9bc4728a88a1)

Reply:
Could you make this into two separate pull requests? One for link and CDN, and one for your translation?

Reply:
I now separately only opened a pull request for `cdn.arsenic.hackclub.com`, but I don't know how to open a separate pull request.

Do I have to wait until the first one gets merged then create a new one?